### PR TITLE
[OBSDEF-12181] [Enhancement] There should be an alert to specify if the pravega components are not ready/initialized. [OBSDEF-12248]add component name into the Remedy table for alerts and issues

### PR DESCRIPTION
**Pull request by mmll**
## Purpose
[OBSDEF-12181](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-12181)
Add 1701 issue: pravega service is not ready
Change 1700 issue file name

## PR checklist
- [ ] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
missing remedies for 1701...

**Comment by mmll** [comment]
> missing remedies for 1701...

Done, thanks Eddie.

**Review by CraneShiEMC — APPROVED** [review]

File: ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml [hunk 1]
```diff
@@ -3,14 +3,11 @@
...
   matchOnList:
-    - matchon:
-        # NORMAL
-        - label: SymptomID
-          value: OBJST-1701
     - matchon:
         # WARNING
         - label: SymptomID
           value: OBJST-1700
+  component: Pravega
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml:10]: is Pravega itself generate this event?
which component container generates this event?
we should list out the name of the component that is generating this event.

File: ecs-cluster/alerts/issues/1701_PRAVEGA_SERVICE_NOT_READY_issue.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+- description: "Pravega Service Status"
+  name: "PRAVEGA_SERVICE_STATUS"
+  defaultAutoClearTimeOut: 60
+  issueCategory: Auto
+  matchOnList:
+    - matchon:
+        # WARNING
+        - label: SymptomID
+          value: OBJST-1701
+  component: Pravega
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/1701_PRAVEGA_SERVICE_NOT_READY_issue.yaml:10]: same

File: ecs-cluster/alerts/remedies/1700_PRAVEGA_CONNECT_remedies.yaml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 - symptomid: OBJST-1700
-  description: Pravega connect status
+  description: Pravega service status
+  component: Pravega
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/1700_PRAVEGA_CONNECT_remedies.yaml:3]: same

File: ecs-cluster/alerts/remedies/1701_PRAVEGA_SERVICE_NOT_READY_issue.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+- symptomid: OBJST-1701
+  description: Pravega service status
+  component: Pravega
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/remedies/1701_PRAVEGA_SERVICE_NOT_READY_issue.yaml:3]: same

**Review by vgupta-mickey — COMMENTED** [review]
Please review all the remedies for each symptom id  and make sure they all are making sense from the customer's point of view. It is up to the developer who knows what remedies are possible for a given problem.

File: ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml (hunk 1 above)
Comment (mmll) [reply ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml:10]: This event is generated by stream lib which is used by ons, objmt or crr components. Does it mean we need to config the component as these 3 components?

Comment (vgupta-mickey) [reply ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml:10]: you can add the multiple components as a string as long the components are defined in  the owner reference document.

File: ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml [hunk 5]
```diff
@@ -3,14 +3,11 @@
...
   matchOnList:
-    - matchon:
-        # NORMAL
-        - label: SymptomID
-          value: OBJST-1701
     - matchon:
         # WARNING
         - label: SymptomID
           value: OBJST-1700
+  component: Stream
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/1700_PRAVEGA_CONNECT_issue.yaml:10]: remove it..we just need to add into remedies

File: ecs-cluster/alerts/issues/1701_PRAVEGA_SERVICE_NOT_READY_issue.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+- description: "Pravega Service Status"
+  name: "PRAVEGA_SERVICE_STATUS"
+  defaultAutoClearTimeOut: 60
+  issueCategory: Auto
+  matchOnList:
+    - matchon:
+        # WARNING
+        - label: SymptomID
+          value: OBJST-1701
+  component: Stream
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/1701_PRAVEGA_SERVICE_NOT_READY_issue.yaml:10]: same

**Review by eddiepavkovic — COMMENTED** [review]
why is there 1420 commits in this PR?  was this created from an older charts repo?
@pgraves5 ^^

**Pull request closed without merging**